### PR TITLE
[teemo] Update to v2.3

### PR DESCRIPTION
**Pull request by winsoft666**
**Describe the pull request**

- #### What does your PR fix?  
Update to v2.3 version.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  <linux, widnows / osx, uwp>, <Yes>

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  <Yes>

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/teemo/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "teemo",
-  "version-string": "2.2",
-  "port-version": 2,
+  "version-string": "2.3",
+  "port-version": 3,
```
Comment (JonLiu1993) [inline ports/teemo/vcpkg.json:4]:
```suggestion
  "version": "2.3",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/teemo/vcpkg.json (hunk 1 above)
Comment (winsoft666) [reply ports/teemo/vcpkg.json]: OK, updated!

**Comment by JonLiu1993** [comment]
@winsoft666 ,LGTM, plesase run command "./vcpkg x-add-version teemo --overwrite-version" and commit again

File: ports/teemo/vcpkg.json (hunk 1 above)
Comment (winsoft666) [reply ports/teemo/vcpkg.json]: > @winsoft666 ,LGTM, plesase run command "./vcpkg x-add-version teemo --overwrite-version" and commit again

OK, updated!

File: versions/t-/teemo.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "d33e668cc0750872a5853c680fa707e12f032e9e",
+      "version": "2.3",
+      "port-version": 0
+    },
+    {
+      "git-tree": "80d77038daf023f9adf68e1b8d9e49797bbbbed8",
+      "version-string": "2.3",
+      "port-version": 3
+    },
```
Comment (JonLiu1993) [inline versions/t-/teemo.json:12]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the version update!

**Pull request merged**